pl_examples/basic_examples/autoencoder.py

```diff
@@ -124,7 +124,7 @@ def predict_dataloader(self):
 
 
 def cli_main():
-    cli = LightningCLI(LitAutoEncoder, MyDataModule, seed_everything_default=1234)
+    cli = LightningCLI(LitAutoEncoder, MyDataModule, seed_everything_default=1234, save_config_overwrite=True)
```

Comment:
this was just for debugging or will you leave that?

Reply:
The CI was breaking a config already existed. Better to force overwriting.